src/pip/__init__.py

```diff
@@ -1,6 +1,6 @@
 from typing import List, Optional
 
-__version__ = "24.3.dev0"
+__version__ = "25.0.dev0"
```

Comment:
is that correct?

Reply:
The release will be cut off of [`551c8de` (#13044)](https://github.com/pypa/pip/pull/13044/commits/551c8deb51175d913d00eb871988448890feb3da) which contains the right version. Our release flow can be bit of an oddball :P